package.json

```diff
@@ -25,7 +25,7 @@
     "build:linux": "yarn run compile && electron-builder --linux --dir -c.productName=Lens",
     "build:mac": "yarn run compile && electron-builder --mac --dir -c.productName=Lens",
     "build:win": "yarn run compile && electron-builder --win --dir -c.productName=Lens",
-    "test": "jest --env=jsdom src $@",
+    "test": "scripts/test.sh",
```

Comment:
This helps support `yarn test <file-name>` only testing one file

Reply:
I don't really like that we split these into yet-another script (worst case: make -> yarn -> bash). How about:

```
    "jest-jsdom": "jest --env=jsdom",
    "test": "yarn run jest-jsdom src",
```

Now you could run custom targets using `yarn jest-jsdom path/to/file.test.ts`.

Reply:
Probably we should't do such changes. If we eliminate `"testEnvironment": "node",` from our `package.json`, tests still works okay and `jsdom` set as default value. So we don't need passing extra vars `--env=jsdom` or
```
/**
 * @jest-environment jsdom
 */
```
declaration in specific files.

Reply:
Ah so we could just remove the "test" script all together then, brilliant!